core/core-impl/src/main/java/ai/timefold/solver/core/api/solver/SolverManager.java

```diff
@@ -98,6 +99,65 @@ static <Solution_, ProblemId_> SolverManager<Solution_, ProblemId_> create(
         return new DefaultSolverManager<>(solverFactory, solverManagerConfig);
     }
 
+    // ************************************************************************
+    // Builder methods
+    // ************************************************************************
+
+    /**
+     * Creates a Builder that allows to customize and submit a planning problem to solve.
+     *
+     * @param problemId never null, a ID for each planning problem. This must be unique.
+     *        Use this problemId to {@link #terminateEarly(Object) terminate} the solver early,
+     * @param problem never null, a {@link PlanningSolution} usually with uninitialized planning variables
+     * @return never null
+     */
+    default SolverJobFactory<Solution_, ProblemId_> builder(ProblemId_ problemId, Solution_ problem) {
+        return builder(problemId, (problemId_) -> problem);
+    }
```

Comment:
Personally, I'd leave the `solution` etc. all the way for the `solve(solution)` and `solveAndListen(solution)`. Keep the number of overloads in the `SolverManager` to a minimum.